.github/workflows/ci.yaml

```diff
@@ -76,13 +75,13 @@ jobs:
 
       - name: Build project
         run: |
-          unity -quit -batchmode -nographics -logFile - -projectPath ${{ env.PROJECT_PATH }} -buildWindows64Player artifacts/build/game.exe | Out-Default
+          unity -quit -batchmode -nographics -logFile - -projectPath samples/unity-of-bugs -buildWindows64Player artifacts/build/game.exe | Out-Default
```

Comment:
isn't it better to have this path as a variable? Why walk away from it?

Reply:
it's actually simpler for me when i see the path directly.
it would make sense if the project path changed, but otherwise it's just clearer this way imo.